### DISCUSSION TITLE
License - Apache or EULA

### DISCUSSION THREAD

**Discussion by hmehra** [ory/keto#1514] [Q&A]
Hi Ory team, 

We are evaluating using self hosted Ory Keto for our RBAC system. But we see there are 2 licenses present in the repo - One is Apache ([here](https://github.com/ory/keto/blob/master/LICENSE)) and the other one is a EULA ([here](https://github.com/ory/keto/blob/master/.releaser/LICENSE.txt)). Can you please let me know which one applies?

**Comment by aeneasr** [comment]
Hi, Apache 2 is correct
